src/Native/Silk.NET.MoltenVK.Native/build/netstandard2.0/Silk.NET.MoltenVK.Native.targets

```diff
@@ -0,0 +1,17 @@
+<Project ToolsVersion="15.0">
+  <PropertyGroup>
+    <_LibMoltenVKPath Condition="'$(RuntimeIdentifier)' == 'maccatalyst-x64' OR '$(RuntimeIdentifier)' == 'maccatalyst-arm64'">$(MSBuildThisFileDirectory)..\..\runtimes\MoltenVK.xcframework\ios-arm64_x86_64-maccatalyst\libMoltenVK.a</_LibMoltenVKPath>
+    <_LibMoltenVKPath Condition="$(RuntimeIdentifier) == 'iossimulator-x64' OR '$(RuntimeIdentifier)' == 'iossimulator-arm64'">$(MSBuildThisFileDirectory)..\..\runtimes\MoltenVK.xcframework\ios-arm64_x86_64-simulator\libMoltenVK.a</_LibMoltenVKPath>
+    <_LibMoltenVKPath Condition="'$(RuntimeIdentifier)' == 'ios-arm64'">$(MSBuildThisFileDirectory)..\..\runtimes\MoltenVK.xcframework\ios-arm64\libMoltenVK.a</_LibMoltenVKPath>
```

Comment:
This is not the standard layout for .NET native binaries. I don't think there's a precedent in .NET where xcframeworks are used. `runtimes/ios-arm64/libMoltenVK.a` should be sufficient.

Reply:
I took this xcframework from https://github.com/KhronosGroup/MoltenVK/releases/tag/v1.2.5 and preferred to leave its structure as is to make it easier to maintain.

Reply:
I left only libraries, without xcframework